<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeControlWidget</class>
 <widget class="QWidget" name="TimeControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
   <item row="0" column="1">
    <widget class="QPushButton" name="pauseButton">
     <property name="text">
      <string>| |</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="startButton">
     <property name="text">
      <string>&gt;</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="restartButton">
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Speed [%]: </string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="timeSlider">
     <property name="minimum">
      <number>-500</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="timeSpinBox">
     <property name="minimum">
      <number>-500</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>timeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>timeSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>timeSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
